Monika After Story/game/script-compliments.rpy

```diff
@@ -888,6 +889,174 @@ label mas_compliment_humor_3:
     m 1hubsu "[humor_quip]"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_compliments_database,
+            eventlabel="mas_compliment_missed21",
+            prompt="I missed you21!",
```

Comment:
A fpyo.
```suggestion
            prompt="I missed you!",
```

Reply:
Resolved in https://github.com/Monika-After-Story/MonikaModDev/pull/7419/commits/95507c54619f07b5712e9c8768214c151988f092.